pyproject.toml

```diff
@@ -25,6 +25,7 @@ dependencies = [
   "libcst == 1.1.0", # 1.2.0+ needs Python 3.9+
   "lxml ~= 5.0",
   "mako==1.1.6",
+  "ninja==1.11.1.1",
```

Comment:
Since the Python stuff is a separate change can you pull it out into a separate commit? (The PR can hold both)

Reply:
done

Reply:
done

Reply:
done